Forms/Main.cs

```diff
@@ -825,7 +827,7 @@ private void ShowDownloadInfo(int epid)
             }
 
             this.SetToolbarButtons(buttons.ToArray());
-            this.SetSideBar(TextUtils.StripDateFromName(info.Name, info.Date), infoText, Model.Episode.GetImage(epid));
+            this.SetSideBar(TextUtils.StripDateFromName(progInfo.Name + " - " + info.Name, info.Date), infoText, Model.Episode.GetImage(epid));
```

Comment:
I'm not sure if this addition belongs in this PR as it looks like it adds the programme name before the episode name on the sidebar.  If you'd like to add this as well, I'd suggest splitting it out as a separate PR.